sources/advanced/Cli.ts

```diff
@@ -405,7 +408,7 @@ export class Cli<Context extends BaseContext = BaseContext> implements MiniCli<C
                     result += `\n`;
                 }
 
-                result += `${this.format(colored).bold(prefix)}${this.getUsageByRegistration(commandClass)}\n`;
+                result += `${this.format(colored).bold(prefix)}${this.getUsageByRegistration(commandClass, {showOptionList: true})}\n`;
```

Comment:
I'm not sure if I should add `showOptionList` option at line 342 or 393 as well. I don't know how I to access those code paths.

Reply:
No it's fine to keep them as they are. The first one is the command listing on the main help page (and we don't want options to be individually there), the second is to print the usage for error messages.